src/views/v2/rsvp/ari/form/guest-template.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * This template renders the RSVP AR form guest template.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp/ari/form/guest-template.php
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+
+?>
+<script
+	class="tribe-tickets__rsvp-ar-form-guest-template"
+	id="tmpl-tribe-tickets__rsvp-ar-form-guest-template-<?php echo esc_attr( $rsvp->ID ); ?>"
+	type="text/template"
+>
+	<div
+		class="tribe-tickets__rsvp-ar-form-guest tribe-common-a11y-hidden"
+		data-guest-number="{{data.attendee_id + 1}}"
+	>
+
+		<?php $this->template( 'v2/rsvp/ari/form/template/title', [ 'rsvp' => $rsvp ] ); ?>
+
+		<?php $this->template( 'v2/rsvp/ari/form/template/fields', [ 'rsvp' => $rsvp ] ); ?>
+
+		<?php $this->template( 'v2/rsvp/ari/form/buttons', [ 'rsvp' => $rsvp ] ); ?>
```

Comment:
I felt adding/removing this on a guest addition/removal basis is more clear than updating the listeners on a "navigation basis". The submit buttons "are only there" when they are actually needed, if not, the markup is there, but they're kind of invisible.

Reply:
While reading on this, I really thing we should avoid printing items we are not goign to show. Avoids confusion from people using filters.

Reply:
The nature of the design/interaction of this block is hiding and showing things. It's like a "tabs" with pagination and validation between them (a form has to be complete and valid to proceed to the next guest). I feel the buttons belong to the place I put them.

Imagine of having a carousel/slider of videos, with a video on each. You have a play and a pause button. You'll want to have those play and pause button on each video instead of having them outside the slider, and assigning their action depending on what's visible. And when the video is playing you'll see the pause button. This is kind of the same.